pro/src/components/pages/Signup/SignupForm/__specs__/SignupForm.spec.jsx

```diff
@@ -179,49 +177,67 @@ describe('src | components | pages | Signup | SignupForm', () => {
       ).toBeInTheDocument()
     })
 
-    it('should enable submit button only when required inputs are filled', () => {
+    it('should enable submit button only when required inputs are filled', async () => {
       // Given the signup form
-      render(
-        <Router history={history}>
-          <SignupForm {...props} />
-        </Router>
-      )
+      await renderSignupForm({ props, storeOverride: store })
+
       const submitButton = screen.getByRole('button', {
         name: /Créer mon compte/,
       })
       // when the user fills required information
-      userEvent.type(
+      userEvent.paste(
         screen.getByRole('textbox', {
           name: /Adresse e-mail/,
         }),
         'test@example.com'
       )
-      userEvent.type(screen.getByLabelText(/Mot de passe/), 'user@AZERTY123')
-      userEvent.type(
+      userEvent.paste(screen.getByLabelText(/Mot de passe/), 'user@AZERTY123')
+      userEvent.paste(
         screen.getByRole('textbox', {
           name: /Nom/,
         }),
         'Nom'
       )
-      userEvent.type(
+      userEvent.paste(
         screen.getByRole('textbox', {
           name: /Prénom/,
         }),
         'Prénom'
       )
-      userEvent.type(
+      userEvent.paste(
```

Comment:
`paste` est destiné à coller depuis le presse-papier, sa signature a changé dans la dernière version de userEvent et son utilisation à la place de `type` empêche la mise à jour de la lib https://github.com/testing-library/user-event/releases/tag/v14.0.0

Il va falloir nettoyer toute la codebase, alors je ne suis pas très favorable à en rajouter, d'autant que cela fonctionne très bien avec input

Reply:
Nous avions décidé de ne plus utiliser `userEvent` a cause de `type` qui pouvait significativement rallonger la durée des tests.
L'utilisation de `paste` permet de tester un trigger unique de changement de valeurs.

Si on ne peu plus utiliser paste, nous pouvons repasser e fireEvent pour les input text.
Sinon, on doit pouvoir utiliser la nouvelle api de paste mise à disposition.

Reply:
si je comprend bien il faudra remplacer 
```javascript
userEvent.paste(myInput, 'string')
```
par 
```javascript
userEvent.paste('myString')
userEvent.paste(myInput)
```
https://github.com/testing-library/user-event/pull/785

Reply:
```js
paste(clipboardData?: DataTransfer|string): Promise<void>
```
  When called without clipboardData, the content to be pasted is read from the Clipboard.

Il semble qu'on puisse garder notre utilisation actuel avec l'ajout de la clef `clipboardData`.
```js
userEvent.paste(monElement, 'mon text')
```
devient
```
userEvent.paste(monElement, {
    clipboardData: {
      getData: () => 'mon text',
    })
```

J'ai vraiment peur que le typing nous trigger plein d'evenement dont nous n'avons pas besoin dans les test.

Reply:
Non je crois qu'on ne peut plus mettre `monElement` comme premier argument

Je ne vois pas trop quel est le problème à utiliser `userEvent.type(` qui est fait pour ça ?

Reply:
Pour répondre à ta question dans slack `hello, tu lis ce que j'ecris dans mes commentaire ?`,  oui je confirme que j'ai bien lu ton commentaire, mais je n'ai peut-être pas compris: pourquoi userEvent.type pose des problèmes de typing ?

Reply:
Nous pouvons en discuter par meet ou sur slack, je pense avoir été claire lorsque j'ai répondu à ta question [la première fois ici](https://github.com/pass-culture/pass-culture-main/pull/1877#discussion_r839639468).

Reply:
Je trouve des issu tel que celle ci : https://github.com/testing-library/user-event/issues/577
Des chaînes d’effet sur des changements de valeurs dans des inputs peuvent allongé de façons significative la durée de nos test.
Nous avons décidé il y à un bon moment d'utilisé paste afin de ne trigger les effet qu'une seul fois lorsque nous n'avons pas besoin de tester un comportement "lettre par lettre".